packages/paste-website/src/pages/components/page-header/index.mdx

```diff
@@ -134,7 +136,7 @@ Default Page Headers should have `space-130` clearspace above them. Place platfo
 
 When the user navigates to an unrelated page, such as a separate product’s marketing page, it might not make sense to use a Breadcrumb in the page header for navigation. Instead, use an anchor with the text “<-- Return to [previous page name]”.
```

Comment:
```suggestion
When the user navigates to an unrelated page, such as a separate product’s marketing page, it might not make sense to use a Breadcrumb in the page header for navigation. Instead, use an anchor with `ArrowBackIcon` and the text “Return to [previous page name]”.
```